generators/docker-compose/__snapshots__/docker-compose.spec.ts.snap

```diff
@@ -1200,9 +1200,9 @@ jhipster:
   },
   "realm-config/keycloak-health-check.sh": {
     "contents": "#!/bin/bash
-exec 3<>/dev/tcp/localhost/9080
+exec 3<>/dev/tcp/localhost/9000
```

Comment:
Did the default port change to 9000? If so, that's not good because it'll create a conflict with the port used for `npm start`.

Reply:
https://www.keycloak.org/2024/06/keycloak-2500-released
<img width="1348" alt="image" src="https://github.com/jhipster/generator-jhipster/assets/9989211/9fb0292e-3841-4ac0-89f2-a1e5552da384">
I will see to change the port